_data/alert.yml

```diff
@@ -0,0 +1 @@
+message: "[OpenSearch 1.2.0 is live 🍾 Try the new observability interace, branding customizer, and more!](/downloads.html)"
```

Comment:
interace -> interface

Otherwise looks great! Thanks!

Reply:
that's what I get for drafting messages in a text editor w/o spell check 🤦‍♂️

Thanks!